pkg/controller/storagecluster/initialization_reconciler.go

```diff
@@ -196,7 +207,13 @@ func (r *ReconcileStorageCluster) ensureExternalStorageClusterResources(instance
 			} else if d.Name == cephRgwStorageClassName {
 				// Set the external rgw endpoint variable for later use on the Noobaa CR (as a label)
 				// Replace the colon with an underscore, otherwise the label will be invalid
-				externalRgwEndpointReplaceColon := strings.Replace(d.Data[externalCephRgwEndpointKey], ":", "_", -1)
+				rgwEndpoint := d.Data[externalCephRgwEndpointKey]
+				if err := checkRGWEndpoint(rgwEndpoint, 30*time.Second); err != nil {
```

Comment:
```suggestion
				if err := checkRGWEndpoint(rgwEndpoint, 5*time.Second); err != nil {
```

A lower 5s timeout should be more than enough to validate the connectivity.

Reply:
Done